opal/admin.py

```diff
@@ -111,14 +119,20 @@ def view_on_site(self, obj):
 
 
 class PatientAdmin(VersionAdmin):
-    list_display = ('__str__', 'patient_detail_link')
+    list_display = ('__str__', 'name', 'identifier', 'patient_detail_link')
 
     search_fields = [
         'demographics__first_name',
         'demographics__surname',
         'demographics__hospital_number'
     ]
 
+    def name(self, obj):
+        return obj.demographics().name
+
+    def identifier(self, obj):
+        return obj.demographics().hospital_number
+
     def patient_detail_url(self, obj):
         return "/#/patient/{0}".format(obj.id)
```

Comment:
This is probably wrong if we're looking at this now... 
You can do reverse / get absolute url I think